.github/renovate.json5

```diff
@@ -606,6 +606,12 @@
       "matchPackageNames": ["corentinth/it-tools"],
       "automerge": true,
     },
+    {
+      "matchDatasources": ["docker"],
+      "matchPackagePrefixes": ["ghcr.io/crazy-max/rtorrent-rutorrent"],
+      "versioning": "regex:^(?<major>\d+)([.-](?<minor>\d+)([._](?<patch>\d+(-r\d+)?)?)?)?$",
```

Comment:
https://regex101.com/r/1iSabd/1

It doesn't match at all

Reply:
appolgies, fixed it and it passed https://regex101.com/r/1iSabd/1 
for  4.2.9-0.9.8_2-0.13.8_1 and for 4.2.9-0.9.8-0.13.8

Reply:
Thats not fixed. each regex group must match a single **number**.

Reply:
I think I'll just close the or and add a request to have someone fix this. I'm not too sure how to properly do that